packages/unpublished-model-encapsulation-test-app/package.json

```diff
@@ -18,6 +18,7 @@
     "test": "ember test --test-port=0"
   },
   "dependencies": {
+    "@ember-data/private-build-infra": "3.16.0-beta.0",
```

Comment:
Do we need `@ember-data/canary-features`

Reply:
followed up with @runspired; this is a transitive dep so we're good to go for here